Docs/README-csharp-v2.md

```diff
@@ -0,0 +1,255 @@
+# Usage examples - C&#35;
+
+## Basics
+
+The PhaxioClient class is the entry point for any Phaxio operation (you'll also need to add ```using Phaxio.V2```).
+
+    var phaxio = new PhaxioV2Client(key, secret);
```

Comment:
Is there a way to keep the class name "PhaxioClient" or just "Phaxio", have it know to use a default version (the latest) and optionally allow the user to add a version as an argument?  Example:

Users v2:
`var phaxio = new Phaxio(key, secret);`

Also uses v2:
`var phaxio = new Phaxio(key, secret, "v2");`

Uses v1:
`var phaxio = new Phaxio(key, secret, "v1");`